Common/src/main/scala/nl/lumc/sasc/biopet/test/MultisampleMapping.scala

```diff
@@ -35,7 +35,7 @@ trait MultisampleMapping extends Pipeline {
 
   def flexiprepShouldRun = !skipFlexiprep.getOrElse(defaultSkipFlexiprep)
 
-  def mappingToGears: Option[Boolean] = Some(false)
+  def mappingToGears: Option[String] = Some("none")
```

Comment:
`Some('none')`?? If even None is a Some, why not just make the type a regular string?